rollup.config.js

```diff
@@ -23,19 +24,21 @@ for (const file of fs.readdirSync(targetDir)) {
 module.exports = functions.map(fn => ({
   input: fn,
   plugins: [
+    resolve({preferBuiltins: false}),
```

Comment:
I'm a tad lost here.... and haven't found the proper way to tame "assert" and "os"

```
(!) Plugin node-resolve: preferring built-in module 'assert' over local alternative at 'H:\spectral\node_modules\assert\build\assert.js', pass 'preferBuiltins: fal
se' to disable this behavior or 'preferBuiltins: true' to disable this warning
(!) Plugin node-resolve: preferring built-in module 'os' over local alternative at 'os', pass 'preferBuiltins: false' to disable this behavior or 'preferBuiltins: 
true' to disable this warning
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
assert (imported by dist\rulesets\oas\functions\oasTypedEnum.js,  assert?commonjs-external)
os (imported by node_modules\better-ajv-errors\node_modules\supports-color\index.js,  os?commonjs-external, node_modules\@babel\highlight\node_modules\supports-col
```

@P0lip Help? 🙏